packages/devtools_app/lib/src/performance/flutter/performance_screen.dart

```diff
@@ -172,4 +243,25 @@ class _PerformanceScreenBodyState extends State<PerformanceScreenBody>
       recordedObject: 'CPU samples',
     );
   }
+
+  void _exportPerformance() {
+    final exportedFile = controller.exportData();
+
+    Notifications.of(context)
+        .push('Successfully exported $exportedFile to ~/Downloads directory');
```

Comment:
add a helper method `successfulExportMessage` to import_export.dart (similar to `attemptingToImportMessage` that takes a file name and returns this string.

use that helper here and in the timeline.

Also add a TODO to find a way to push the notification logic into the export controller.
we structure TODOs as "// TODO(username): todo message"